src/audio/mixin_mixout/Kconfig

```diff
@@ -1,12 +1,8 @@
 # SPDX-License-Identifier: BSD-3-Clause
 
-config COMP_ARIA
-        bool "ARIA component"
-        default n
-        depends on IPC_MAJOR_4
-        help
-          Select for Automatic Regressive Input Amplifier Module component
-	  ARIA applies variable gain into incoming signal.
-	  Applied gain is in range <1, 2 power attenuation>
-	  Currently ARIA introduces gain transition and algorithmic
-	  latency equal to 1 ms.
+config COMP_MIXER
```

Comment:
Hmm, it doesn't seem correct to have two COMP_MIXER definitions with different semantics. This works now as Zephyr and XTOS builds use different Kconfigs (and this is getting slowly fixed in #8606 ), but this seems very confusing. Maybe better to just fix the extra ARIA defintiion and leave COMP_MIXER intact.